sources/plugins/scg/scgwindow.cpp

```diff
@@ -456,6 +456,42 @@ void SCgWindow::onExportImage()
     }
 }
 
+void SCgWindow::exportToPng(int index)
+{
+
+    SCgExportImage exportImage;
+    QString selectedFilter;
+
+    QString imageName = currentFileName();
+    QString fileName = QCoreApplication::applicationDirPath() + "/massExport/";
+    QDir().mkdir(fileName);
+    if (imageName.isEmpty())
+    {
+        fileName.append(QString::QString::number(index));
+    }
+    else
+    {
+        QStringList sl = imageName.split("/");
+        fileName.append(sl[sl.size()-1]);
+    }
+    if (fileName.length() > 0)

```

Comment:
```suggestion
    if (!fileName.isEmpty())
```
For more clarity & to conform with lines 468, 481.